public/js/src/articleListModel.js

```diff
@@ -0,0 +1,29 @@
+(function(exports) {
+  function ArticleList() {
+    this.list = []
+  }
+  ArticleList.prototype.listAll = function() {
+    return this.list
+  }
+  ArticleList.prototype.add = function(headline) {
+    let article = new Article(headline)
+    this.list.push(article)
+    article.id = this.list.indexOf(article)
+  }
+
+  ArticleList.prototype.getData = function() {
+    let self = this
+    fetch('http://news-summary-api.herokuapp.com/guardian?apiRequestUrl=http://content.guardianapis.com/travel')
+      .then(response => response.json())
+      .then(data => {
+        let results = data.response.results;
+        results.forEach((item, i) => {
+          self.add(item.webTitle)
+          self.list[i].url = item.apiUrl
```

Comment:
I would add a class here called article, to get to code that looks like this:
```js
results.forEach((item) => {
          let article = new Article(item.webTitle, item.apiUrl)
          self.add(article)
});
```
Then each article is responsible for its own title and URL, and could also produce its own html.

If you don't want another class, it may still be cleaner to do:
```js
results.forEach((item) => {
          let article = {title: item.webTitle, url: item.apiUrl)
          self.add(article)
});
```